lib/variant-model.js

```diff
@@ -45,17 +47,42 @@ _.extend(Variant.prototype, {
     return this;
   },
 
-  respondWithFile: function(fileName) {
+  respondWithFile: function(options) {
+    options = _.extend({
+      variant_id: this.id(),
+      path: this._route.path(),
+    }, options);
+
     return this.respondWith(function(request, reply) {
-      var self = this;
-      fileName = fileName.replace(/\{([^\}]+)\}/g, function(match, token) {
-        var val = request.params[token];
-        if (!val) {
-          val = self.state(token);
+      var smocks = require('./index');
```

Comment:
can you add a comment that we are requiring `smocks` here on purpose so we can lazy load the options - just so we don't forget in the future and accidentally refactor.